e2e/components/standards/svap/svap.spec.js

```diff
@@ -45,6 +47,16 @@ describe('the svap component', () => {
       await action.save();
       await browser.waitUntil(async () => (await component.getSvaps()).length > initialCount);
       await expect(await (await component.svapTable).getText()).toContain(version);
+      await page.editStandards(version);
+      await (await component.version).clearValue();
+      await (await component.version).setValue(newVersion);
+      await action.save();
+      await expect(await (await component.svapTable).getText()).toContain(newVersion);
```

Comment:
I think we need a "wait" in here before expecting our results. I agree that this probably works, but as a general practice we should wait for something to change, then expect our new something to happen. In the add/delete cases we wait for the table length to change. I recognize that won't work here, but maybe we could wait for the old name to not be found in the table? 

Maybe something like:

```suggestion
      await action.save();
      await browser.waitUntil(async () => (await component.svapTable).getText().not.toContain(version);
      await expect(await (await component.svapTable).getText()).toContain(newVersion);
```